packages/cojson/src/sync.ts

```diff
@@ -830,7 +829,7 @@ export class SyncManager {
     return this.local.storage?.waitForSync(id, this.local.getCoValue(id));
   }
 
-  waitForSync(id: RawCoID, timeout = 30_000) {
+  waitForSync(id: RawCoID, timeout = 60_000) {
```

Comment:
Increasing the timeout to make it more reasonable.

In the past we needed this because the whole system was less predictable, but with the recent improvements we can be more confident on not getting "stuck".


Thinking about removing it completely, or making it even longer just as a safeguard to stuck errors.